### ISSUE TITLE
The results of Cora

### ISSUE THREAD

**Issue by sxu-yaokx** [chennnM/GCNII#15]
Hi! I have reproduced your paper recently, it's an interesting paper. But I get an 83.6% accuracy for Cora with your code, the parameters are the same as you provided.  I am curious about the reason.

I'm looking forward to your reply! Thank you!

**Comment by SwaksharDeb** [comment]
Hi, there. I also run their code several times but always get 83% validation accuracy for cora dataset. I read their paper and found that they got 85.5% validation accuracy for cora dataset. I did not change anything in their code. Moreover, I recheck the hyperparameter values from their paper. I found the same hyperparameter values being used in their code also. But I am not able to reproduce their result in spite of running their code several times.
